### ISSUE TITLE
error when command contain newline

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [tymm/zsh-directory-history#12]
Paste in the terminal a command with a newline at the end and get this error

```
Traceback (most recent call last):
  File "/usr/local/bin/dirhist", line 104, in <module>
    print "\n".join(get_commands_in_directory(directory))
  File "/usr/local/bin/dirhist", line 48, in get_commands_in_directory
    directory_in_history, command = get_dir_and_command_previous_version(line.strip())
  File "/usr/local/bin/dirhist", line 22, in get_dir_and_command_previous_version
    return (parts[0], parts[1])
IndexError: list index out of range
```

to temporarily solve i delete the blank line in .directory_history

**Comment by tymm** [comment]
Thanks!
I will look into it in the next few days.

**Comment by tymm** [comment]
I pushed the fix for this issue into the develop branch.
I will test it for a couple of days before I will merge it into master.

The new version will however use a new format for the `.directory_history` file.
You will have to start a new empty `.directory_history` file or modify your existing one (append "\0\n" at the end of every line).

**Comment by tymm** [comment]
This issue should be fixed with [v1.1.0](https://github.com/tymm/zsh-directory-history/releases/tag/v1.1.0).
However this new release introduces a new history format and will not support old history files anymore.
You can manually change your existing history file to be supported by the new version by appending (before the newline) `\0` (the null character) at every line in your `.directory_history` file.
